app/src/components/SightingList.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
```

Comment:
[fix] `import * as Reacat from "react";`

See https://twitter.com/dan_abramov/status/1308739731551858689?lang=en

Fix for all instances in other files as well.